tests/autobahn/Dockerfile.aiohttp

```diff
@@ -0,0 +1,18 @@
+FROM python:3.9.1
+
+ENV DEBIAN_FRONTEND=noninteractive
+RUN \
+    apt-get update && \
+    apt-get -y install --no-install-recommends bash && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN \
+    wget -q -O /usr/bin/wait-for-it https://raw.githubusercontent.com/vishnubob/wait-for-it/master/wait-for-it.sh && \
+    chmod +x /usr/bin/wait-for-it
+
+COPY ./ /src
+
+WORKDIR /src
+RUN pip install --no-cache-dir -e .
+RUN pip install attrs
```

Comment:
Why is this necessary?

Reply:
^

Reply:
@anesabml this thread doesn't have any response from you.

Reply:
@anesabml ^

Reply:
Reminder to reply here.

Reply:
the `attrs` packaged is used [here](https://github.com/aio-libs/aiohttp/blob/4d58400477331704132cc9427d66699e5815ee25/aiohttp/web_protocol.py#L23) I think this is why the original author used `dataclass` instead

Reply:
Alright. So it looks like https://github.com/aio-libs/aiohttp/pull/5284 attempted to wipe any use of attrs across the repo but @asvetlov missed this file. This probably needs to be taken care of: https://github.com/aio-libs/aiohttp/issues/5806.

Please add a code comment next to this line explaining the need.

Reply:
```suggestion
# NOTE: The attrs lib is necessary because it's
# NOTE: still a de-facto runtime dependency of
# NOTE: aiohttp. Although it shouldn't be anymore
# NOTE: after the effort to get rid of it.
# Refs:
# * https://github.com/aio-libs/aiohttp/issues/5806
# * https://github.com/aio-libs/aiohttp/pull/5679/files#r654428147
# * https://github.com/aio-libs/aiohttp/pull/5284
RUN pip install attrs
```